### PR TITLE
Show alert link to Troubleshooting Guide when reload fails

### PR DESCRIPTION
**Pull request by varjolintu**
Shows a link to the Troubleshooting Guide every third time Reload fails when the popup is open. Hiding the alert prevents it appearing again.
Screenshot 2022-04-17 at 13 16 14

### CODE REVIEW

File: keepassxc-browser/popups/popup.js [hunk 1]
```diff
@@ -87,6 +96,12 @@ const sendMessageToTab = async function(message) {
         statusResponse(await browser.runtime.sendMessage({
             action: 'reconnect'
         }));
+
+        // Shows the Troubleshooting Guide alert every third time Reload button is pressed when popup is open
+        if (reloadCount > 2) {
+            reloadCount = 0;
+        }
+        reloadCount++;
```
Comment (droidmonkey) [inline keepassxc-browser/popups/popup.js:104]: Need to reset this when a successful connection is made.

**Review by droidmonkey — APPROVED** [review]

File: keepassxc-browser/popups/popup.js [hunk 2]
```diff
@@ -25,6 +27,13 @@ function statusResponse(r) {
         if (r.showGettingStartedGuideAlert) {
             $('#getting-started-guide').show();
         }
+
+        if (r.showTroubleshootingGuideAlert && reloadCount === 2) {
```
Comment (droidmonkey) [inline keepassxc-browser/popups/popup.js:31]: Recommend a `>= 2` here

**Review by droidmonkey — APPROVED** [review]

File: keepassxc-browser/popups/popup.js (hunk 2 above)
Comment (varjolintu) [reply keepassxc-browser/popups/popup.js:31]: Ok. Let's fix that too.

File: keepassxc-browser/popups/popup.js [hunk 3]
```diff
@@ -87,6 +98,12 @@ const sendMessageToTab = async function(message) {
         statusResponse(await browser.runtime.sendMessage({
             action: 'reconnect'
         }));
+
+        // Shows the Troubleshooting Guide alert every third time Reload button is pressed when popup is open
+        if (reloadCount > 2) {
+            reloadCount = 0;
+        }
```
Comment (droidmonkey) [inline keepassxc-browser/popups/popup.js:105]: Wait a sec this doesn't make sense. If you attempt another reload it hides the banner?

Comment (varjolintu) [reply keepassxc-browser/popups/popup.js:105]: This happens after the `statusResponse()` where the comparison mention above is.

**Pull request merged**